pages/docs/tutorials/cli-documantation-validation.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Validate AsyncApi documents with CLI commands'
+description: 'This tutorial will teach you how to validate AsynApi document using CLI commands'
+weight: 115
+---
+## Introduction
+
+When it comes to APIs, validation is inevitable because it helps you keep them well-defined. Validation means testing an API to see if it meets the required functionality, performance, security, and quality standards. It is important to validate an API to ensure that it functions properly. This tutorial will teach you how to validate your AsyncAPI documents using your CLI (Command Line Interface).
+
+## Background context
+
+An AsyncAPI document is a file that defines and describes the various components of a message-based or event-driven API. Event-driven APIs are a type of asynchronous API that allows consumers to subscribe to events of interest. It adheres to an architecture in which events are fired or triggered. These triggers occur when a producer detects a state change caused by a user's device or an action taken by the user. The producer is an application that sense state changes and produces these changes as messages.
+
+AsyncAPI document enables people or machines communicating with one another to comprehend the capabilities of an event-driven API without requiring access to the source code, documentation, or network traffic inspection.
+The document allows developers to define the structure of their API, the format of the API, the events to which consumers can subscribe, and the response they receive when these events are triggered or an action is taken. AsyncAPI documents are represented as JSON objects that follow the JSON.
+
+## Installation Guide
```

Comment:
hey! where is the tutorial? 😂 Please steps showing how to validate an AsyncAPI document with the CLI.

also, you can find the install instructions for AsyncAPI CLI in it's repo: https://github.com/asyncapi/cli/blob/master/docs/installation.md

that said, you should prob use our fragment that was created exactly for this: https://github.com/asyncapi/website/blob/master/assets/docs/fragments/cli-installation.md

You can add that fragment with the following snippet...

```
import CliInstallation from '../../../assets/docs/fragments/cli-installation.md' 

<CliInstallation/>
```

Reply:
The tutorial will be included soon. I like to get feedback as I work on the documentation bit by bit to avoid going off track. Thank you for the feedback and assistance @alequetzalli